haystack/components/generators/chat/hugging_face_api.py

```diff
@@ -21,20 +21,19 @@
 @component
 class HuggingFaceAPIChatGenerator:
     """
-    A Chat Generator component that uses Hugging Face APIs to generate text.
+    Completes chats using Hugging Face APIs.
 
-    This component can be used to generate text using different Hugging Face APIs with the ChatMessage format:
+    HuggingFaceAPIChatGenerator uses the [ChatMessage](https://docs.haystack.deepset.ai/docs/data-classes#chatmessage)
+    format for input and output. Use it to generate text with Hugging Face APIs in the
+    ChatMessage format:
```

Comment:
```suggestion
    format for input and output. Use it to generate text with Hugging Face APIs:
```

Since we're already mentioning ChatMessage as output in previous sentence